modules/core/config/Config.swift

```diff
@@ -2,25 +2,57 @@ import Foundation
 
 /**
  * AeroGear Mobile Services Configuration
- * Responsible for parsing and processing configuration for individual services.
+ * Entrypoint for parsing and processing configuration for individual services.
+ *
+ * This class is responsible for providing configurations for services of certain types.
+ * Configuration is stored in single json file that contains multiple individual service configurations and metadata.
+ * Service developers can query configuration for certain types. See top level core interface for more information
+ *
+ * Usage:
+ *
+ *     `config['myServiceType']`
  */
-public class Config {
+public class ServiceConfig {
 
     let configFileName: String
     var config: MobileConfig
 
-    init(_ configFileName: String = "mobile-services") {
+    public init(_ configFileName: String = "mobile-services") {
```

Comment:
You want to init use outside of project ? If yes use open instead of public

Reply:
Idea is that this class is created and managed internally by component.